chain/db/db.go

```diff
@@ -50,18 +54,37 @@ func (b *BandDB) RollBack() {
 	b.tx = nil
 }
 
-func (b *BandDB) HandleEvent(eventName string, attributes map[string]string) {
-	switch eventName {
+func (b *BandDB) HandleTransaction(tx auth.StdTx, txHash []byte, logs sdk.ABCIMessageLogs) {
```

Comment:
We may also need other information like block time, block height. Might even be better to just pass Context here?

Reply:
We need info about that Send context here or save context in bandDB state?